.github/CODEOWNERS

```diff
@@ -1,4 +1,7 @@
 Gemfile         @theforeman/packaging
-Rakefile.dist	@theforeman/packaging
+Rakefile.dist   @theforeman/packaging
 bundler.d/*.rb  @theforeman/packaging
 package.json    @theforeman/packaging
+package.json    @theforeman/ui-ux
```

Comment:
I think this doesn't work. https://help.github.com/articles/about-codeowners/ says it should be `package.json @theforeman/packaging @theforeman/ui-ux`

Reply:
I would also add webpack configuration files and the code under /scripts/ that handle the packaging / webpack integration.